docker/k8s/tcs_config.toml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Intel Corporation
```

Comment:
duplicated config file.

Reply:
This is to avoid corrupting the one in `config` under the project root.

Later on more setting may diverge between this one and its original in the `config` directory.

Reply:
I would suggest not to duplicate file unless absolutely necessary. Maintenance will be difficult with duplicate files. Existing config file can add kubernetes specific sections if required though I don't find any kubernetes section now.

Reply:
The duplicate config file has been removed as https://github.com/hyperledger-labs/trusted-compute-framework/pull/47/commits/17002ab20a329bdedaca59b9750c36c482756e71